### PR TITLE
Remove deploy data from gamedata, and ignore development data 

### LINKED ISSUE #32: writing development deploy addresses to gamedata.json causes merge conflicts
When a level is deployed during development, a level's "deployment_development" member is written to. "deployment_ropsten" is supposed to be unique but "deployment_development" is not, and is currently causing merge conflicts.

Consider extracting development deployment addresses to a separate file that can be gitignored.

### PR DESCRIPTION
**Pull request by ajsantander**
Fixes #32 

Ready to merge.

Changes where the app looks for deployed level addresses and how the deployment script stores these addresses in the json files.

All deployment addresses where removed from `gamedata.json`. Each level has a `deployId` that is used as a key to retrieve the last deploy address from specific per-network `deploy.<network_name>.json` files. The development file `deploy.development.json` is added to `.gitignore`. As a result, development deployed addresses are removed completely from vcs.

The deployment script was updated, as well as the UI that reads in to the deployment addresses stored in the json files.

Gameplay needs to be tested.

### CODE REVIEW

File: scripts/deploy_contracts.js [hunk 1]
```diff
@@ -26,20 +28,25 @@ async function exec() {
 
   await initWeb3()
 
+  // Retrieve deployment data for the active network.
+  try {
+    deployData = require(`../gamedata/deploy.${constants.ACTIVE_NETWORK.name}.json`)
+  }
+  catch(err){
+    deployData = {};
```
Comment (elopio) [inline scripts/deploy_contracts.js]: shouldn't you halt and catch fire here? What does it mean to not have deployData?

File: src/containers/Sidebar.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ class Sidebar extends React.Component {
             // Created
             const creationDate = moment(level.created)
             const ago = moment.duration(moment().diff(creationDate)).asDays() || 0
-						console.log(`Level ${level.name}, created days ago: ${ago}`);
+						// console.log(`Level ${level.name}, created days ago: ${ago}`);
```
Comment (elopio) [inline src/containers/Sidebar.js]: why to keep this statement commented out? If it's not needed, it's better to remove it.

File: src/middlewares/loadGamedata.js [hunk 3]
```diff
@@ -9,22 +9,24 @@ export default store => next => action => {
...
-        level.deployedAddress = level[`deployed_${constants.ACTIVE_NETWORK.name}`][0]
+        level.deployedAddress = deployData[level.deployId]
         level.idx = i;
         levelsOut.push(level);
       }
-      action.ethernautAddress = data.ethernaut[`deployed_${constants.ACTIVE_NETWORK.name}`][0]
+      action.ethernautAddress = deployData.ethernaut
       action.levels = levelsOut;
-      // console.log(`LEVELS DATA (out):`, levelsOut, data.ethernautAddress)
+      // console.log(`LEVELS DATA (out):`, levelsOut, action.ethernautAddress)
```
Comment (elopio) [inline src/middlewares/loadGamedata.js]: same comment as above, commented out code should be removed.

File: scripts/deploy_contracts.js (hunk 1 above)
Comment (ajsantander) [reply scripts/deploy_contracts.js]: No, the idea is that if an object is not obtained from that address, initialize an empty object.

Comment (ajsantander) [reply scripts/deploy_contracts.js]: Aaaaactually... if the file isn't there in the first place, it doesn't work. So I'm going to have to fix this before merge.

**Pull request merged**